internal/util/ptr/ptr.go

```diff
@@ -0,0 +1,29 @@
+package ptr
```

Comment:
I just discovered that these functions have now been extracted from the Kubernetes repo and are available as a package here:
 * https://github.com/kubernetes/utils/tree/master/pointer as
 * `import "k8s.io/utils/pointer"`